### ISSUE TITLE
Packer fails to copy chef cookbooks on the latest version of Docker for Mac (1.12.0-rc2)

### ISSUE THREAD

**Issue by poblahblahblah** [mitchellh/packer#3642]
Packer is failing to upload chef cookbook files from the local filesystem into the container. This results in the following error:

```
==> Some builds didn't complete successfully and had errors:
--> docker: Error uploading cookbooks: Upload failed with non-zero exit status: 1
```

In debug mode it is always a different file that it takes issue with. I've sanitized the output slightly (replacing our domain with example.com). Keep in mind this is Docker for Mac, not docker-machine or boot2docker.

Packer Version: 0.10.1
Host Platform: OSX 10.11.5
Debug log: https://gist.github.com/poblahblahblah/895efe4b4d7c5b152065a2c6b4e34da0
Docker version: Docker for Mac (docker version: 1.12.0-rc2, build 906eacd, experimental)
Example template: https://gist.github.com/poblahblahblah/c4b87e5392bec0a19c992e67ac1dd8a9

**Comment by berniedurfee-ge** [comment]
Same issue with Docker for Mac version 1.12.0-rc4

**Comment by mwhooker** [comment]
I think this will be resolved by #4186. After that gets merged, please open a new issue if you continue to see this
